p2p/simulations/http.go

```diff
@@ -318,6 +319,13 @@ func (s *Server) StopNetwork(w http.ResponseWriter, req *http.Request) {
 	w.WriteHeader(http.StatusOK)
 }
 
+// StopNetwork stops all nodes in the network
```

Comment:
This comment should describe `ResetNetwork`

Reply:
ok, fixed